examples/deadline/All-In-AWS-Infrastructure-Basic/python/cdk.json

```diff
@@ -1,3 +1,18 @@
 {
-  "app": "python -m package.app"
+  "app": "python -m package.app",
+  "context": {
```

Comment:
Hmm, that's a good point. CDK Did change their default values. I have copied over the settings from the `cdk.json` that gets generated when using `cdk init`.

This shouldn't break any existing deployments that don't have these flags, but if they don't have the `defaultKeyPolicies: true` flag, the change to the aws-kms construct `Key` where I've removed the deprecated `trustAccountIdentities` field would affect the `Key`'s policy.

Reply:
That sounds like a breaking change to me, then. We didn't have that context value set in our examples before, so I wouldn't expect customers to have it set either. 

`trustAccountIdentities` is deprecated but not removed, right? I'd think that it would be a breaking change in a stable API for the CDK to remove it in the 1.xx line. What's the effect of us leaving in the usage of the deprecated field? Just a warning? If so, then I think that we leave setting that field in place... we can add these context values as well, but I think that we want to be more careful with breaking changes.

Reply:
Yeah, I think that the warning about being removed in the next major release is fine. That's CDK 2.x. The update of RFDK to that version will be a bit more involved, so it makes sense to me to address this at that time.